.infrastructure/iam.tf

```diff
@@ -148,6 +148,14 @@ data "aws_iam_policy_document" "ecs_task_execution" {
     resources = ["*"]
   }
 
+  statement {
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogGroup"
+    ]
+    resources = ["*"]
+  }
+
```

Comment:
Avoid to define the resource `*` unless strictly necessary, follow always the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).

I suggest to use the [CloudWatch Log Group created by default](https://github.com/terraform-aws-modules/terraform-aws-ecs/blob/master/docs/README.md#container-definition:~:text=The%20default%20behavior%20of%20the%20container%20definition%20module%20is%20to%20create%20the%20CloudWatch%20log%20group%20on%20behalf%20of%20users%20to%20ensure%20it%20can%20be%20fully%20managed%20through%20Terraform) by the [service module](https://github.com/pagopa/developer-portal/blob/9217615/.infrastructure/ecs.tf#L52-L79) if the `container_definitions` is defined or using a Log Group created via IaC, instead of having ECS create them separately. This ensures that all resources are connected to the Infrastructure as Code (IaC). By doing so, we can define the retention policy, which I'm uncertain can be achieved if ECS creates the resources, attach the tag and so on.

Reply:
as agreed, the "strapi-cms" log group will be created in Terraform and this log group will be used in the taks-def